controller/transfer/project_transfer_manager.py

```diff
@@ -650,6 +650,12 @@ def import_file(
                 finished_at=embedding_item.get(
                     "finished_at",
                 ),
+                platform=embedding_item.get(
+                    "platform",
+                ),
+                model=embedding_item.get(
+                    "model",
+                ),    
```

Comment:
maybe a comment that because of security reasons API key aren't exported so we don't "forget" about it

Reply:
done